test/cex/cex_nonlinear_arith.c

```diff
@@ -13,8 +16,8 @@ extern int nd_int(void);
 
 int main(int argc, char**argv) {
   int x = nd_int();
-  __VERIFIER_assume (x >= 1);
-  __VERIFIER_assume (x <= 100);  
+  assume (x >= 1);
```

Comment:
`__VERIFIER_assume` was correct. `assume` is expanded into unknown external function because `seahorn/seahorn.h` is not included

Reply:
`seahorn/seahorn.h` is included on line 13 though

Reply:
I am looking at diff on github, so must have missed that line